politeiawww/api/cms/v1/v1.go

```diff
@@ -30,6 +30,8 @@ const (
 	RouteSupportOpposeDCC    = "/dcc/supportoppose"
 	RouteNewCommentDCC       = "/dcc/newcomment"
 	RouteDCCComments         = "/dcc/{token:[A-z0-9]{64}}/comments"
+	RouteApproveDCC          = "/admin/approvedcc"
```

Comment:
Why have this as two separate routes instead of a single route for changing the DCC status?  It looks like parts of it are even coded as if its a single route.

https://github.com/decred/politeia/pull/982/files#diff-00707343bc75c93602e7ea1744066afcR1000